packages/libraries/cli/src/base-command.ts

```diff
@@ -119,18 +120,18 @@ export default abstract class extends Command {
     return requestId ? requestId.split(',')[0].trim() : undefined;
   }
 
-  registryApi(registry: string, token: string) {
-    return getSdk(
-      new GraphQLClient(registry, {
-        headers: {
-          Accept: 'application/json',
-          'User-Agent': `hive-cli/${this.config.version}`,
-          Authorization: `Bearer ${token}`,
-          'graphql-client-name': 'Hive CLI',
-          'graphql-client-version': this.config.version,
-        },
-      }),
-    );
+  async registryApi(registry: string, token: string) {
+    const response = await fetch(registry, {
+      headers: {
+        Accept: 'application/json',
+        'User-Agent': `hive-cli/${this.config.version}`,
+        Authorization: `Bearer ${token}`,
+        'graphql-client-name': 'Hive CLI',
+        'graphql-client-version': this.config.version,
+      },
+    });
+
+    return response.json();
```

Comment:
What if we get 5XX or 4XX error? :)

Reply:
fixed, now it's the same as graphql-request (throwing an error). Also added handling of `response.errors` with status 200